source/voice_control/troubleshooting.markdown

```diff
@@ -104,6 +104,15 @@ To fix the URL, follow these steps:
    ![Create alias for entity name](/images/assist/local_url.png)
 
 
+### Missing Media Source
+
+If you are using YAML configuration make sure the media source is present:
```

Comment:
This only applies if one isn't using the default_config integration (and thus has removed it manually). Let's make sure we mention that and reference it.